packages/front-end/components/Settings/NewExperimentChecklistItem.tsx

```diff
@@ -0,0 +1,145 @@
+import CreatableSelect from "react-select/creatable";
+import { ChecklistTask } from "back-end/types/experimentLaunchChecklist";
+import Field from "../Forms/Field";
+import { ReactSelectProps } from "../Forms/SelectField";
+
+type AutoChecklistOption = {
+  value: string;
+  label: string;
+  propertyKey: "hypothesis" | "screenshots" | "description" | "project" | "tag";
+};
+
+export default function NewExperimentChecklistItem({
+  experimentLaunchChecklist,
+  setExperimentLaunchChecklist,
+  newTaskInput,
+  setNewTaskInput,
+}: {
+  experimentLaunchChecklist: ChecklistTask[];
+  setExperimentLaunchChecklist: (checklist: ChecklistTask[]) => void;
+  newTaskInput: ChecklistTask;
+  setNewTaskInput: (task: ChecklistTask | undefined) => void;
+}) {
+  const autoChecklistOptions: AutoChecklistOption[] = [
+    {
+      value: "Add a descriptive hypothesis for this experiment",
+      label: "Add a descriptive hypothesis for this experiment",
+      propertyKey: "hypothesis",
+    },
+    {
+      value: "Upload a screenshot for each variation of the experiment",
+      label: "Upload a screenshot for each variation of the experiment",
+      propertyKey: "screenshots",
+    },
+    {
+      value: "Add a description for this experiment",
+      label: "Add a description for this experiment",
+      propertyKey: "description",
+    },
+    {
+      value: "Add this experiment to a project",
+      label: "Add this experiment to a project",
+      propertyKey: "project",
+    },
+    {
+      value: "Add atleast 1 tag to this experiment",
+      label: "Add atleast 1 tag to this experiment",
+      propertyKey: "tag",
+    },
+  ];
+
+  function addNewTask(newTaskInput: ChecklistTask) {
+    setExperimentLaunchChecklist([...experimentLaunchChecklist, newTaskInput]);
+    setNewTaskInput(undefined);
+  }
+
+  return (
+    <div className="pt-5 pb-2">
+      <h4>New Task</h4>
+      <div className="p-3 border rounded">
+        <label>Task</label>
+        <CreatableSelect
+          className="pb-3"
+          options={autoChecklistOptions.filter((option) => {
+            return !experimentLaunchChecklist.some(
+              (index) => index.task === option.value
+            );
+          })}
+          placeholder="Choose from pre-defined tasks or create your own custom task"
+          onChange={(option: AutoChecklistOption) => {
+            setNewTaskInput({
+              task: option.value,
+              completionType: "auto",
+              propertyKey: option.propertyKey,
+            });
+          }}
+          onCreateOption={(inputValue) => {
+            setNewTaskInput({
+              task: inputValue,
+              completionType: "manual",
+            });
+          }}
```

Comment:
Is this safe to use change vs create to differentiate here?  What if they create an item, select something else, and then select their custom item again?

Reply:
@jdorn From my testing - onChange is only fired when the user selects from the pre-defined options list, whereas onCreate is only fired when the user generates their own option and hits either the `enter` key or the `Create XYZ` cta within the dropdown.

I've confirmed that things work as expected in the case outlined above (the user created an item, selects a pre-defined option, and then goes back to the custom item.

One important distinction to call out, we're not adding the user-generated items to the `autoChecklistOption` array when we call `onCreate`. Since an option can only be added to the checklist a single time. So that allows us to use the `onChange` vs `onCreate` options to differentiate if it's a manual or automatic item.